add-drop-ubuntu-python support v22.12 and v23.02 (#310)
rapidsai/docs: _notices/rsn0025.md

---
layout: notice
parent: RAPIDS Support Notices
grand_parent: RAPIDS Notices
nav_exclude: true
notice_type: rsn
# Update meta-data for notice
notice_id: 25 # should match notice number
notice_pin: true # set to true to pin to notice page
title: "Support for Ubuntu 22.04 in v23.02"
notice_author: RAPIDS Ops
notice_status: In Progress
notice_status_color: yellow
# 'notice_status' and 'notice_status_color' combinations:
#   "Proposal" - "blue"
#   "Completed" - "green"
#   "Review" - "purple"
#   "In Progress" - "yellow"
#   "Closed" - "red"
notice_topic: Platform Support Change
notice_rapids_version: "v22.12"
notice_created: 2022-11-21
# 'notice_updated' should match 'notice_created' until an update is made
notice_updated: 2022-11-21
---

## Overview

RAPIDS is planning GA support for `Ubuntu 22.04` in its `v23.02` release. `Ubuntu 22.04` will replace the `Ubuntu 18.04` OS distributions for future RAPIDS releases. Experimental support will begin in `v23.02` nightlies.

## Impact

> `Ubuntu 18.04` users are encouraged to migrate to `Ubuntu 22.04` if possible and provide feedback to the RAPIDS libraries as we develop `v23.02`.

Reviewer: ```suggestion
`Ubuntu 18.04` users are encouraged to migrate to a supported version of Ubuntu as soon as possible.
```

---
layout: notice
parent: RAPIDS Support Notices
grand_parent: RAPIDS Notices
nav_exclude: true
notice_type: rsn
# Update meta-data for notice
notice_id: 25 # should match notice number
notice_pin: true # set to true to pin to notice page
title: "Support for Ubuntu 22.04 in v23.02"
notice_author: RAPIDS Ops
notice_status: In Progress
notice_status_color: yellow
# 'notice_status' and 'notice_status_color' combinations:
#   "Proposal" - "blue"
#   "Completed" - "green"
#   "Review" - "purple"
#   "In Progress" - "yellow"
#   "Closed" - "red"
notice_topic: Platform Support Change
notice_rapids_version: "v22.12"

> notice_created: 2021-11-21

Reviewer 2: ```suggestion
notice_created: 2022-11-21
```

# 'notice_updated' should match 'notice_created' until an update is made
notice_updated: 2022-11-21
---

## Overview

> With the depracation of `Ubuntu 18.04` announced in [RSN 24](/notices/rsn0024), development

Reviewer 2: ```suggestion
With the deprecation of `Ubuntu 18.04` announced in [RSN 24](/notices/rsn0024), development
```

> effort has been redirected to support `Unbuntu 22.04` in `v23.02` nightly releases.

Reviewer 2: ```suggestion
effort has been redirected to support `Ubuntu 22.04` in `v23.02` nightly releases.
```

## Status

> Dropping support for Ubuntu 18.04 is an ongoing effort.

Reviewer 2: Should this status be about bringing 22.04 online, instead of dropping 18.04?

## Impact

> Users should consider switching to any of the following supported operating systems for any `rapidsai/rapidsai` image:

Reviewer 2: I don't think it makes sense to tell users to switch to a supported platform in a message about bringing a new platform online. (Also, the same note as above on Rocky Linux 8.)

> ## Overview
> 
> With the deprecation of `Ubuntu 18.04` announced in [RSN 24](/notices/rsn0024), development
> effort has been redirected to support `Ubuntu 22.04` in `v23.02` nightly releases.
> 
> ## Status
> 
> Dropping support for Ubuntu 18.04 is an ongoing effort.
> 
> ## Impact
> 
> Users should consider switching to any of the following supported operating systems for any `rapidsai/rapidsai` image:
>   - Ubuntu 20.04
>   - Ubuntu 22.04
>   - CentOS 7
>   - CentOS 8

Reviewer 2: ```suggestion
## Overview

Development effort will begin supporting `Ubuntu 22.04` in `v23.02`. `Ubuntu 22.04` will replace the `Ubuntu 18.04` OS distributions for future RAPIDS releases. Experimental support will begin in `v23.02` nightlies. GA support for `Ubuntu 22.04` is expected for `v23.02` stable.

## Impact

`Ubuntu 18.04` users are encouraged to migrate to `Ubuntu 22.04` if possible and provide feedback to the RAPIDS libraries as we develop `v23.02`.
```
